javagen/src/main/java/com/azure/autorest/mapper/ClientMapper.java

```diff
@@ -132,13 +133,16 @@ public Client map(CodeModel codeModel) {
             codeModel.getOperationGroups().stream().flatMap(og -> og.getOperations().stream())
                 .map(o -> parseHeader(o, settings)).filter(Objects::nonNull));
 
-        final List<ClientModel> clientModels = autoRestModelTypes
+        List<ClientModel> clientModelsFromCodeModel = autoRestModelTypes
             .distinct()
             .map(autoRestCompositeType -> Mappers.getModelMapper().map(autoRestCompositeType))
             .filter(Objects::nonNull)
             .distinct()
             .collect(Collectors.toList());
-
+        // append some models not from CodeModel
+        final List<ClientModel> clientModels = Stream.concat(clientModelsFromCodeModel.stream(), ClientModels.getInstance().getModels().stream())
```

Comment:
Confirm: This will add `MultipartFileDetails` model only currently?

Reply:
It could be multiple this kind of "FileDetails".

If the field is `image`, it be `ImageFileDetails`. e.g. https://github.com/Azure/autorest.java/tree/015cbeecb103ce84152d93cb4ce5827b772f26c7/typespec-tests/src/main/java/com/payload/multipart/models

Srikanta wants different name, to avoid different SDK conflict on this model name.

It should be same class, if moved to azure-core.

Reply:
I see, will this add other classes unrelated to multipart?

Reply:
As far as I remember, we didn't have "other classes", till this.

And test does not have new class.

Reply:
And I can delete this, if FileDetails class get to azure-core

Reply:
ok, I see.